cirq-core/cirq/transformers/target_gatesets/sqrt_iswap_gateset.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Target gateset used for compiling circuits to √iSWAP + 1-q rotations + measurement gates."""
```

Comment:
Why measurement gates ?

Reply:
Measurement gates will be "accepted" by the gatesets if they are part of the circuit. So, if you have a `PauliMeasurementGate` in your circuit and you call `cirq.optimize_for_target_gateset(c, gateset = cirq.SqrtIswapTargetGateset())`, it will decompose the pauli measurement into single qubit rotations + `cirq.Measurement` gates because pauli measurements are not part of the gateset but measurement gates are.